src/views/OrderList.vue

```diff
@@ -185,13 +205,78 @@ const isSaveButtonDisabled = computed(() => {
                 </button>
             </div>
             </footer>
+            <div class="order-details-notification">
+                <!-- NOTIFICATION -->
+                <p v-if="notification?.msg" :class="{notification: true, success:notification.success, failed:!notification.success}">
+                    <small>{{ notification?.msg }}</small>
+                </p>
+            </div>
+        </form>
+    </div>
+    </article>
+    <article class="update-order-modal" v-else>
+        <header>
+            <a href="#close"
+            aria-label="Close"
+            class="close"
+            data-target="order-detail"
+            @click="closeModal">
+            </a>
+            <h5 class="modal-title">Order Details</h5>
+            <h6>#SLN: {{ currentOrder?.sln }}<span class="edit-icon"><i class="fa-solid fa-pen-to-square fa-xl" style="color: #c0ca33;"  @click.prevent="editBtnEnabled = !editBtnEnabled" ></i></span></h6>
```

Comment:
Same here.

```suggestion
                <h6>#SLN: {{ currentOrder?.sln }}<a href="#" @click.prevent="editBtnEnabled = !editBtnEnabled" class="edit-icon"><i class="fa-solid fa-pen-to-square fa-xl" style="color: #c0ca33;" ></i></a></h6>
```

Also, remove inline styling and create a css class.

Reply:
done!